go.mod

```diff
@@ -8,14 +8,15 @@ require (
 	github.com/censync/go-validator v1.0.0
 	github.com/corestario/kyber v1.6.1-0.20201110123848-0eac241a9f75
 	github.com/fatih/color v1.9.0
-	github.com/gin-gonic/gin v1.7.2
+	github.com/gin-gonic/gin v1.7.2 // indirect
```

Comment:
There are unused imports in go.mod, I suggest to run `go mod tidy` to clean the file.